### PR TITLE
Ticket 4

### PR DESCRIPTION
**Pull request by mikesealey**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+import Topics from "./Topics"
+import Users from './Users'
+import Search from './Search'
+
+const Header = () => {
+    console.log("In the header")
```
Comment (NRMSMITH) [inline src/components/Header.jsx:9]: make sure to delete these!!

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
   return (
     <>
-  <h1>NC News</h1>
+      <Header/>
+      <Routes>
+        <Route path="/" element={<Home/>}></Route>
+        <Route path="/topics" element={<Topics/>}></Route>
+        <Route path="/users" element={<Users/>}></Route>
+        <Route path="/search" element={<Search/>}></Route>
+      </Routes>
```
Comment (NRMSMITH) [inline src/App.jsx:23]: you haven't built these routes yet, so they shouldn't really appear on this ticket and branch!

File: src/components/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react"
+import axios from "axios"
+import { Link } from "react-router-dom"
+
+const Home = () => {
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        axios.get("https://nc-news-x8sg.onrender.com/api/articles", {})
```
Comment (NRMSMITH) [inline src/components/Home.jsx:9]: this should be extracted into a utils folder - utilise axios create in there :)

File: src/components/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            setArticles(data)
+        })
+    }, [])
+
+    return (
+        <>
+        <div className="body">
+            {articles.map((article) => {
+                return (
+                        <Link to={`/${article.article_id}`}>
```
Comment (NRMSMITH) [inline src/components/Home.jsx:21]: Could this be extracted into an ArticleCard?

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
I've left you a few things that could do with changing, to make your code a lot more functional as this becomes a heavier repo. Also, I would suggest changing the font colour - it's a bit difficult on the eyes. Finally, make sure that your design is responsive (i.e. when I make the screen smaller, I can still see everything -> I shouldn't ever have to scroll sideways to see what's going on!) 

Please resubmit once done 😄

**Review by itskatherine — COMMENTED** [review]
Nice stuff Mike, a few minor points to look over, once you have you're good to merge 🙂

**Pull request merged**